src/stores/useHBSStore.js

```diff
@@ -0,0 +1,40 @@
+import { defineStore } from 'pinia'
+import { fetchAgentKycLevel, registrationFetchJurisdictions } from '../services/hbs'
+
+const makeUseHBSStore = ({ useHoloStore }) => {
```

Comment:
```suggestion
const makeUseHbsStore = ({ useHoloStore }) => {
```

Not blocking, but I prefer this capitalization

Reply:
renamed to makeUseHbsStore